extensions/wikia/ContentReview/tests/ContentReviewApiControllerTest.php

```diff
@@ -33,6 +34,45 @@ public function testSubmitPageForReview( $params, $textExpected, $message ) {
 		$this->contentReviewApiControllerMock->submitPageForReview();
 	}
 
+	/**
+	 * Tests if the updateReviewsStatus method performs all necessary checks to validate
+	 * a request and check a user's permissions to perform the action.
+	 * @param bool $wasPosted
+	 * @param bool $matchEditToken
+	 * @param bool $isAllowed
+	 * @param string $exception A name of a class of an expected exception
+	 * @dataProvider updateReviewsStatusDataProvider
+	 */
+	public function testUpdateReviewsStatus( $wasPosted, $matchEditToken, $isAllowed, $exception ) {
+		/* @var \WikiaRequest $requestMock */
+		$requestMock = $this->getMockBuilder( '\WikiaRequest' )
+			->disableOriginalConstructor()
+			->setMethods( [ 'wasPosted' ] )
+			->getMock();
```

Comment:
Yup, I could simplify it, but because the original `__constructor` requires the first parameter to be an `array` the actual line is: ```$requestMock = $this->getMock( '\WikiaRequest', [ 'wasPosted' ], [ [] ] );```

Thanks!